### PR TITLE
Added MDIS example

### PR DESCRIPTION
**Pull request by jessemapel**

### CODE REVIEW

File: knoten/csm.py [hunk 1]
```diff
@@ -125,6 +121,13 @@ def generate_latlon_boundary(camera, boundary, semi_major=3396190, semi_minor=33
            List of altitude values
     '''
 
+    if not semi_major:
+        ellipse = csmapi.SettableEllipsoid.getEllipsoid(camera)
+        semi_major = ellipse.getSemiMajorRadius()
+    if not semi_minor:
+        ellipse = csmapi.SettableEllipsoid.getEllipsoid(camera)
+        semi_minor = ellipse.getSemiMinorRadius()
+
```
Comment (jessemapel) [inline knoten/csm.py:131]: I may need to back this change out until the SettableEllipsoid class is wrapped in swig.

**Comment by jlaura** [comment]
This is going to sound dumb, but it is super nice to see some of those 'convenience' functions getting used in conjunction with GDAL.

**Review by jlaura — APPROVED** [review]

**Comment by jlaura** [comment]
I'll leave this open for a bit for others to check out, but :+1: from here. This can merge at any time.

**Review by thareUSGS — COMMENTED** [review]
This is on me. And not really a big deal, but in the MDIS_DEM notebook, the radius should be sphere. I need to update the script which created WKT string from these published IAU radius values (note the IAU2015_Mean col).  Actually J-Christophe has already fix his major re-write for the script, I just need to run it again.
https://github.com/USGS-Astrogeology/GDAL_scripts/blob/master/OGC_IAU2000_WKT_v2/naifcodes_radii_m_wAsteroids_IAU2015.csv

Basically, when a body is defined as a triaxial, the IAU defines a best-fit sphere to be used for map projections. Thus simply change your WKT projection string to:
```
GEOGCS["Mercury 2015",DATUM["D_Mercury_2015",SPHEROID["Mercury_2015_IAU",2439400.0,0]],PRIMEM["Reference_Meridian",0],UNIT["Degree",0.0174532925199433],AUTHORITY["IAU2015","19900"]]
```
